pom.xml

```diff
@@ -81,7 +81,7 @@ under the License.
     <mavenVersion>4.0.0-beta-3</mavenVersion>
 
     <guiceVersion>6.0.0</guiceVersion>
-    <junitVersion>5.10.2</junitVersion>
+    <junitVersion>5.12.2</junitVersion>
```

Comment:
it is also defined in parent ...

Reply:
Property removed so dependency management from parent is taken